spec/models/piece_spec.rb

```diff
@@ -50,4 +50,73 @@
       king.errors.clear
     end
   end
+  describe "#move_to!" do
+    context "when the tile is empty" do
+      it "moves to the coordinates" do
+        board = create(:game)
+        board.pieces.delete_all
+        king = game.kings.first
```

Comment:
change this to king = King.create(x_position: 1, y_position: 1, game_id: board.id)